QEfficient/cloud/execute.py

```diff
@@ -38,7 +39,29 @@ def main(
     model_hf_path = hf_download(repo_id=model_name, cache_dir=cache_dir, allow_patterns=["*.json"])
     tokenizer = AutoTokenizer.from_pretrained(model_hf_path, use_cache=True, padding_side="left")
 
-    cloud_ai_100_exec_kv(tokenizer=tokenizer, qpc=qpc_path, device_id=devices, prompt=prompt)
+    if inputs_file_path is not None:
+        try:
+            prompt = []
+            with open(inputs_file_path, "r") as file:
+                for line in file:
+                    prompt.append(line.strip())
+        except FileNotFoundError:
```

Comment:
Can we add one more except to handle errors that are not `FileNotFoundErrors`, like let's say currupt file or `keyboardinterrupt` error?
Or remove this try except and let the program fail with `FileNotFound` error message?